src/main.rs

```diff
@@ -59,7 +60,18 @@ fn process_matches<'a>(
         .trim()
         .split(' ');
 
-    Ok((cargo_cmd_iter, cmd_iter))
+    // `values_of` returns an `Option` which we then turn into a iterator and flatten
+    let custom_envs = matches
+        .values_of("env")
+        .into_iter()
+        .flatten()
+        // FIXME: Currently invalid env vars are ignored, should they rather make an error?
+        .filter_map(|s| {
+            let mut ss = s.split('=');
+            Some((ss.next()?, ss.next()?))
```

Comment:
`str::find` and a slice would be better here -- this could potentially lose part of the string with input like `"asdf=meh=blarg"`, which SHOULD parse as `"asdf" -> "meh=blarg"`.

Reply:
Ah that is a good point! 😄